### ISSUE TITLE
refactor: clean up accessibility flew code

### ISSUE THREAD

**Issue by avevotsira** [avevotsira/dsc#112]
figures without role = none

**Comment by avevotsira** [comment]
Figure elements that are used to display images should have a "role" attribute that equals to "none" and the image provide itself should provide the description using an "alt" attribute.

**Comment by avevotsira** [comment]
aria-describedby and aria-labeledby attributes should be connected by ID to an element that either has text description or an "aria-label" attribute.

**Comment by avevotsira** [comment]
![image](https://github.com/avevotsira/dsc/assets/100404669/5633ef56-aee3-456c-80bf-ee1de88995ca)
